aioftp/client.py

```diff
@@ -547,13 +722,26 @@ def list(self, path="", *, recursive=False):
 
             >>> stats = await client.list()
         """
-        class AsyncClientLister(AsyncListerMixin):
+        class AsyncLister(AsyncListerMixin):
 
             async def _new_stream(cls, local_path):
 
                 cls.path = local_path
-                command = str.strip("MLSD " + str(cls.path))
-                return await self.get_stream(command, "1xx")
+                str_path = " " + str(cls.path)
+
+                cls.parse_line = self.parse_mlsx_line
+
+                try:
+                    command = str.strip("MLSD" + str_path)
+                    return await self.get_stream(command, "1xx")
+
+                except errors.StatusCodeError as e:
+
+                    if e.received_codes[-1] == "500":
+
+                        cls.parse_line = self.parse_list_line
+                        command = str.strip("LIST" + str_path)
+                        return await self.get_stream(command, "1xx")
```

Comment:
Made a fatal mistake in that code, check again. The only way I can think of to put the fallback on the same level as `try` is to set an variable in `except` and add an if after it to test if an exception is caught. I think that's not a good way to do it, but perhaps you were thinking something else?